_1_navigation.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation</title>
+    <style rel="stylesheet" href="stylesheets/_1_style.css"></style>
+</head>
+<body>
+    <nav class="nav menu breadcrumbs">
+        <ol>
+            <li class="nav__item nav__item--home"><a href="#">Home</a></li>
+            <li class="nav__item"><a href="#">Projects</a></li>
+            <li class="nav__item"><a href="#">Project Nero</a></li>
+        </ol>
+    </nav>
+
+    <nav class="nav menu">
+        <ul>
+            <li class="nav__item"><a href="#">Dashboard</a></li>
+            <li class="nav__item"><a href="#">Team</a></li>
+            <li class="nav__item"><a href="#">Projects</a></li>
+            <li class="nav__item"><a href="#">Calendar</a></li>
+            <li class="nav__item"><a href="#">Documents</a></li>
+            <li class="nav__item"><a href="#">Reports</a></li>
+        </ul>
+    </nav>
```

Comment:
не исп. классы можно было убрать, вообще правильней так, вы же описывается 2 разных компонента

```suggestion
    <nav class="breadcrumbs">
        <ol class="breadcrumbs__list">
            <li class="breadcrumbs__item breadcrumbs__item--home"><a class="breadcrumbs__link" href="#">Home</a></li>
            <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="#">Projects</a></li>
            <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="#">Project Nero</a></li>
        </ol>
    </nav>

    <nav class="menu">
        <ul class="menu__list">
            <li class="menu__item"><a class="menu__link" href="#">Dashboard</a></li>
            <li class="menu__item"><a class="menu__link" href="#">Team</a></li>
            <li class="menu__item"><a class="menu__link" href="#">Projects</a></li>
            <li class="menu__item"><a class="menu__link" href="#">Calendar</a></li>
            <li class="menu__item"><a class="menu__link" href="#">Documents</a></li>
            <li class="menu__item"><a class="menu__link" href="#">Reports</a></li>
        </ul>
    </nav>
```